src/elfpy/pricing_models/base.py

```diff
@@ -205,7 +205,10 @@ def calc_spot_price_from_reserves(
             "pricing_models.calc_spot_price_from_reserves: ERROR: "
             f"expected share_reserves > 0, not {market_state.share_reserves}!",
         )
+        # s = y + c*z
```

Comment:
Please add a TODO here. In general $s \neq y + cz$ and we'll want to update this soon to use `s = market_state.lp_reserves`

Reply:
added TODO